src/unstract/sdk/llm.py

```diff
@@ -50,6 +52,10 @@ def __init__(
         self._adapter_instance_id = adapter_instance_id
         self._llm_instance: LlamaIndexLLM = None
         self._usage_kwargs = usage_kwargs
+        self._capture_metrics = capture_metrics
```

Comment:
Are we really using _capture_metrics anywhere? If not, we could remove this var...

Reply:
Is this an attempt to follow some guideline to keep the instance variables cached for fututr use? Because I see other constructor vars also being stored below.

Reply:
@gaya3-zipstack  It is being used in the `capture_metrics` decorator.